bee-message/src/error/packable.rs

```diff
@@ -0,0 +1,195 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+pub use crate::{
+    input::InputUnpackError,
+    output::{
+        OutputIdUnpackError, OutputUnpackError, SignatureLockedAssetPackError, SignatureLockedAssetUnpackError,
+        SignatureLockedSingleUnpackError,
+    },
+    payload::{
+        data::{DataPackError, DataUnpackError},
```

Comment:
just use `data::*Error` to include all `Error` types that it finds.











okay, that was a joke, but I kinda want that here xD